src/tools/sdk/archive/index.md

```diff
@@ -28,11 +28,19 @@ Stable channel builds are tested and approved for production use.
 
 {% include_relative _archives_table.html channel="stable" %}
 
+## Beta channel
+
+Beta channel builds are preview builds for the stable channel.
+We recommend testing, but not releasing, your apps against beta
+to preview new features, or test compatibility with future releases.
```

Comment:
```suggestion
to preview new features or test compatibility with future releases.
```

Reply:
Should we add anything like "Beta channel builds are not suitable for production use." (to match dev channel)?

Reply:
Yup, done and done